### DISCUSSION TITLE
如何不花一分钱搭建 Umami 统计工具

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#8] [Announcements]
# 如何不花一分钱搭建 Umami 统计工具

好兄弟的博客数据统计被我看到了，不用爬梯子就能看，而且可以公示，界面简洁支持中文，非常符合我的需求，话不多说，开整！

https://zexwoo.blog/posts/tutorials/build-umami/

**Comment by AprDeci** [comment]
谢谢你

**Comment by crisqyxw** [comment]
感谢博主

**Reply by crisqyxw** [reply]
哎，登陆不了，控制台报Internal Server Error🤣

**Reply by crisqyxw** [reply]
[解决方案](https://github.com/umami-software/umami/discussions/1542)

**Comment by kerncos** [comment]
发现一个更简单的方法：[umami](https://guangzhengli.com/blog/zh/how-to-integrate-umami-for-free-to-blog-site/)

**Reply by kerncos** [reply]
不行，有问题，请忽略>.<

**Comment by xyhcode** [comment]


**Reply by ZexWoo** [reply]
我只能保证你严格按照我的方法做能成功。如果你不计划白嫖，可以找找别的方法。

**Comment by syybi** [comment]
用Vercel的Postgres Database数据库岂不是更好？
环境变量填`DATABASE_URL`，值为为Postgres Database数据库字段（格式如下）
```
postgresql://default:xxxxxxxx@cxcxcxcxcxc.aws.neon.tech:5432/verceldb?sslmode=require
```

**Comment by xialaup** [comment]
我觉得还有一个更坑的，就这程序，没有设置累积统计多少天，或者清除陈旧统计的功能，日积月累数据库爆炸，我在2年前在neon免费数据库搭建 才没用几天，数据库2g了 绝👍 直接卸载了
